Code/Modules/effects/BaseEffect.cpp

```diff
@@ -1,5 +1,5 @@
 #include "BaseEffect.h"
-#include <stdlib.h>
+#include <cstdlib>
```

Comment:
all of these include changes are not really relevant to the gcc930 support, and would probably be best left out of this PR..